### PR TITLE
Template

### PR DESCRIPTION
**Pull request by RosaMoran**
@yildiztugba  i wrote last question now we have 10, but if you inspect `the console` i just saw 2 question not 10

### CODE REVIEW

File: client/src/business-logic/initialize-state.js [hunk 1]
```diff
@@ -3,5 +3,5 @@
 import { load } from '../data-access/load.js';
 
 export const initializeState = async () => {
-  await load('../../../data/numbers.json', import.meta);
+  await load('../../../data/quiz.json', import.meta);
```
Comment (yildiztugba) [inline client/src/business-logic/initialize-state.js:6]: Can you check the path in here with template branch? This was old file. I changed path.

**Review by yildiztugba — CHANGES_REQUESTED** [review]

File: client/src/business-logic/initialize-state.js (hunk 1 above)
Comment (lauraramirez220612) [reply client/src/business-logic/initialize-state.js:6]: Yes Tugba,  it is like Samir did today in class.  Can I merge it?

Comment (yildiztugba) [reply client/src/business-logic/initialize-state.js:6]: AA, yes you are right.

Comment (yildiztugba) [reply client/src/business-logic/initialize-state.js:6]: But I realized that @RosaMoran made pr from template to master? Is it correct?

Comment (lauraramirez220612) [reply client/src/business-logic/initialize-state.js:6]: I think is correct.

Comment (RosaMoran) [reply client/src/business-logic/initialize-state.js:6]: @yildiztugba  yes i  made the `PR` you can see it in `template branch` and also the path for  `line 6` of the file `initialize-state.js`  it same that @samir wrote, today i'll start to in UI of `selecctio` part if someone wants help me, that can be great 
have a nice day ladies

Comment (RosaMoran) [reply client/src/business-logic/initialize-state.js:6]: ooh i forgot, this part i'll put it inside of  `data access` 
i'll make a push from the branch `02-startquiz`

**Comment by yildiztugba** [comment]
I will close this pr, cause Rosa made a new one for that.

**Pull request closed without merging**